gef-ui/server/index.js

```diff
@@ -90,7 +90,8 @@ app.use('/', routes);
 
 app.use('/assets', express.static(path.join(__dirname, '..', 'public')));
 
-app.use((err, req, res) => {
+// eslint-disable-next-line no-unused-vars
+app.use((err, req, res, next) => {
```

Comment:
can just remove the `next` param?

Reply:
@IgnatG ?

Reply:
`next` is not being used here, but it's been added - and lint rule added so no point in having it

Reply:
unfortunately, we have to leave it here. For some reason it's throwing errors somewhere else if we don't have it

Reply:
this is our error handler btw... which is not ideal at the moment (it can be improved), but we have to change it later on

Reply:
OK i see 👍